apps/admin-gui/src/app/shared/components/dialogs/bulk-invite-members-dialog/bulk-invite-members-dialog.component.html

```diff
@@ -0,0 +1,81 @@
+<div class="{{theme}}">
+  <h1 mat-dialog-title>{{'DIALOGS.BULK_INVITE_MEMBERS.TITLE' | translate}}</h1>
+  <mat-spinner *ngIf="loading" class="ml-auto mr-auto"></mat-spinner>
+  <div *ngIf="!loading" class="dialog-container" mat-dialog-content>
+    <div *ngIf="state === 'input'">
+      <div *ngIf="!data.groupId" class="font-italic">
+        {{'DIALOGS.BULK_INVITE_MEMBERS.DESCRIPTION_VO' | translate}}
+      </div>
+      <div *ngIf="data.groupId" class="font-italic">
+        {{'DIALOGS.BULK_INVITE_MEMBERS.DESCRIPTION_GROUP' | translate}}
+      </div>
+      <mat-form-field class="w-100 pt-2">
+        <mat-label>{{'DIALOGS.BULK_INVITE_MEMBERS.LANGUAGE' | translate}}</mat-label>
+        <mat-select [(value)]="currentLanguage">
+          <mat-option
+            *ngFor="let lang of languages"
+            value="{{lang}}"
+            >{{'SHARED_LIB.LANGUAGES.'+lang | uppercase | translate}}</mat-option
+          >
+        </mat-select>
+      </mat-form-field>
+      <p [innerHTML]="'DIALOGS.BULK_INVITE_MEMBERS.HINT' | translate"></p>
+      <mat-form-field class="pt-2 flex-container">
+        <mat-label>{{'DIALOGS.BULK_INVITE_MEMBERS.INSERT_HERE'| translate}}</mat-label>
+        <textarea
+          cols="50"
+          class="md-textarea form-control"
+          [formControl]="invitedMembers"
+          required
+          matInput
+          placeholder="{{'DIALOGS.BULK_INVITE_MEMBERS.PLACEHOLDER' | translate}}"
+          rows="8">
+        </textarea>
+        <mat-error *ngIf="invitedMembers.hasError('required')">
+          {{'DIALOGS.BULK_INVITE_MEMBERS.NAMES_ERROR' | translate}}
+        </mat-error>
+        <mat-error *ngIf="invitedMembers.hasError('invalidFormat')">
+          {{'DIALOGS.BULK_INVITE_MEMBERS.ERROR_FORMAT'| translate}}:
```

Comment:
nitpick- if there are too many values inputted, the message can be confusing. maybe different alert should be used in that case.